test/sql/order/test_limit_percent.test

```diff
@@ -0,0 +1,137 @@
+# name: test/sql/order/test_limit_percent.test
+# description: Test LIMIT keyword {% | PERCENT}
+# group: [order]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+CREATE TABLE test (a INTEGER, b INTEGER);
+
+statement ok
+INSERT INTO test VALUES (11, 22), (12, 21), (13, 22), (14, 32), (15, 52)
```

Comment:
Can we add some tests where we test LIMIT% with more types, in particular:

* Strings
* Lists
* Structs
* NULL Values

Reply:
Okay, I will add it